applications/jupyterhub/src/harness_jupyter/harness_jupyter/jupyterhub.py

```diff
@@ -45,9 +45,41 @@ def change_pod_manifest(self: KubeSpawner):
                     #       cpu_limit: 3
                     #       mem_guarantee: 4G
                     #       mem_limit: 8G
-                    if 'spawnerExtraConfig' in harness['jupyterhub']:
-                        for k, v in harness['jupyterhub']['spawnerExtraConfig'].items():
+                    #       node_selector:             # optional node pool for these pods
+                    #         - key: ch/nodepool
+                    #           operator: In           # In, NotIn, Exists, DoesNotExist. Gt, and Lt.
+                    #           values: pool-jupyter
+                    #           matchPurpose: require  # require | prefer | ignore
+                    spawnerExtraConfig = harness['jupyterhub'].get('spawnerExtraConfig')
+                    if spawnerExtraConfig:
+                        for k, v in spawnerExtraConfig.items():
                             setattr(self, k, v)
+                        node_selectors = spawnerExtraConfig.get('node_selectors')
```

Comment:
here is `node_selectors`, the doc above says `node_selector`. Should also be in camelCase to match the general k8s spec convention. Need to also document it in the jupyterhub application readme

Reply:
fixed the type, thanks for catching

Reply:
@filippomc about the camelCase: the other jupyter configs aren't camelCase e.g. `cpu_guarantee`